### PR TITLE
Upgrade to MSW 2.0

### PR DESCRIPTION
**Pull request by ghengeveld**
Uses a modified version of `msw-storybook-addon` for now, until it's been updated to support MSW 2.0.

<details>
  <summary>📦 Published PR as canary version: <code>0.0.110--canary.136.3023bab.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.110--canary.136.3023bab.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.110--canary.136.3023bab.0
  ```
</details>

### CODE REVIEW

File: .storybook/mswDecorator.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
+"use strict";
```
Comment (ghengeveld) [inline .storybook/mswDecorator.js:1]: I straight up copied this from the `dist` of `msw-storybook-addon`, after updating it to support MSW 2.0.

Really the only thing that's changes is the `require('msw/browser')` which used to be `require('msw')`.

**Review by tmeasday — APPROVED** [review]
LGTM

File: .storybook/mswDecorator.js (hunk 1 above)
Comment (yannbf) [reply .storybook/mswDecorator.js:1]: you could just copy/paste the typescript file instead, it would look cleaner and more understandable!

Comment (ghengeveld) [reply .storybook/mswDecorator.js:1]: It wouldn't work due to the way msw-storybook-addon is compiled differently than our project.

**Pull request merged**